app/views/venues/show.html.erb

```diff
@@ -4,4 +4,5 @@
   <%= link_to @venue.website, @venue.website, class: "website", target: "_blank" %>
   <p class="venue-address"><%= @venue.address %></p>
   <p class="venue-capacity">Capacity: <%= @venue.capacity %></p>
+  <p class="venue-t-accessible"><%= @venue.display_t_accessible %></p>
```

Comment:
Awesome. Thanks for the clarification. We were trying too hard to get things out of the view.